docs/src/main/sphinx/connector/row-decoding.fragment

```diff
@@ -0,0 +1,323 @@
+Row decoding
+^^^^^^^^^^^^
+
+For key and message, a decoder is used to map message and key data onto table
+columns.
+
+The connector contains the following decoders:
+
+* ``raw`` - message is not interpreted; ranges of raw message bytes are mapped to table columns.
+* ``csv`` - message is interpreted as comma separated message, and fields are mapped to table columns.
+* ``json`` - message is parsed as JSON, and JSON fields are mapped to table columns.
+* ``avro`` - message is parsed based on an Avro schema, and Avro fields are mapped to table columns.
+* ``protobuf`` - message is parsed based on a Protobuf schema, and Protobuf fields are mapped to table columns.
```

Comment:
@nevillelyh Seems `ProtobufDecoderFactory` is added to `DecoderModule` which gets reused by Redsis and Kinesis as well - do you know if it'll work for those as well or is the implementation coupled to Kafka only?

Reply:
It should assuming we're reading Protobuf encoded `byte[]` from Redis & Kinesis just like Kafka. Although don't think we have tests covering reading protos from Redis & Kinesis.

Reply:
Thanks, I'll create an issue/add some tests - at-least for Redis.